packages/app/src/components/Layout/Layout.js

```diff
@@ -42,10 +45,52 @@ const getLayoutByType = (type, props) => {
     return <Layout {...props} />
 }
 
-const Layout = props => getLayoutByType(props.type)
+const Layout = props => {
+     const onDragEnd = result => {
+        const { source, destination } = result;
+
+        if (!destination) {
```

Comment:
in which scenario is `!destination` true? any case for `!source`?
if so, suggestion: `!destination || !source`

Reply:
I think is when you drop a Draggable outside a Droppable, in that case nothing should happen.
The source on the other hand, should always be present at drag end, because it's the Draggable element you are moving.